ktor-http/common/src/io/ktor/http/Url.kt

```diff
@@ -55,10 +55,9 @@ public class Url internal constructor(
     }
 
     public val encodedQuery: String by lazy {
-        if (parameters.isEmpty()) {
-            return@lazy ""
-        }
         val queryStart = urlString.indexOf('?') + 1
+        if (queryStart == 0) return@lazy ""
```

Comment:
can you do the same change for fragment?

Reply:
sure